src/common/PkmImage/PkmImage.tsx

```diff
@@ -32,18 +32,16 @@ const PkmImage = ({ dexNum, form, name, shiny = false }: PkmImageProps) => {
   };
 
   useEffect(() => {
-    loadImage(imgSrc)
+    loadImage(baseImgSrc + getImage(dexNum, form))
```

Comment:
I would extract the string building to it's own line and add some validation to both to ensure that they hold data and are indeed strings, otherwise you could just create a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to ensure the string type.